supported_kafka.version

```diff
@@ -0,0 +1,2 @@
+2.8.1
```

Comment:
You have 3 `.version` files. But 2 of them are really a single version, one of them has multiple versions. Should this be distinguished in the file name?

Reply:
I can add suffix `versions` to the Kafka version file